lib/interceptOpen.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * interceptOPen.c
+ *
+ *  Created on: Oct 15, 2016
+ *      Author: ujay
+ */
+#define _GNU_SOURCE
+#define _GNU_SOURCE
+#include<dlfcn.h>
+#include<stdio.h>
+#include<stdlib.h>
+#include<sys/stat.h>
+#include<unistd.h>
+#include"../include/cwrapHeader.h"
+
+
+int (*real_openat)(int fd1,const char* pathname1, int flags1)=NULL;
+int open(const char *pathname,int mode){
```

Comment:
See comments from `interceptAccess.c`. Also, we could probably combine all of these wrapper functions into a single C file: they are all quite small, quite similar and require basically the same header files.